### PR TITLE
Fixes borg floaters

### LINKED ISSUE #2193: You can drag borgs on tiles like they were on pools
the dragging borgs outside code doesn't check if the borg's actually in a pool, minor issue

### PR DESCRIPTION
**Pull request by Kokojos**
Fixes #2193
Tested on a live server.

### CODE REVIEW

**Comment by dreamcrab** [comment]
? wat

File: code/game/objects/structures/pool.dm [hunk 1]
```diff
@@ -24,13 +24,14 @@
...
-				T.Exited(M)
-				M.forceMove(src)
-				user << "<span class='notice'>You get out of the pool.</span>"
+			var/turf/T = get_turf(M)
+			if(istype(T, /turf/simulated/pool))
+				M.visible_message("<span class='notice'>[M] begins to float.", \
+					"<span class='notice'>You start your emergency floaters.</span>")
+				if(do_mob(user, M, 20))
+					T.Exited(M)
+					M.forceMove(src)
```
Comment (Carbonhell) [inline code/game/objects/structures/pool.dm:32]: doesn't forcemove call exited already?

**Comment by Badmex** [comment]
Seems like a fix, does it need Headinput? Either way yeah +1/3

**Comment by Kokojos** [comment]
Needs one head imput or a codecheck. IIRC

**Comment by Carbonhell** [comment]
> or a codecheck
> what are you saying,the codecheck is always required, and head input is not needed for bugfixes

File: code/game/objects/structures/pool.dm (hunk 1 above)
Comment (Carbonhell) [inline code/game/objects/structures/pool.dm:32]: ain't gonna merge till i get answered

**Comment by Nebyab** [comment]
sure

File: code/game/objects/structures/pool.dm (hunk 1 above)
Comment (Kokojos) [inline code/game/objects/structures/pool.dm:32]: It does. So I guess il remove them.

**Comment by ArcLumin** [comment]
What'd you do to piss travis off

**Comment by Badmex** [comment]
@Kokojos fix travis

**Comment by Kokojos** [comment]
Travis fixed ready to merge.

**Comment by Carbonhell** [comment]
merging

**Pull request merged**